virttest/iscsi.py

```diff
@@ -256,6 +298,11 @@ def __init__(self, params, root_dir):
             self.emulated_image = emulated_image
             self.device = "device.%s" % os.path.basename(self.emulated_image)
 
+        # optimize the `create file` command
+        self.create_cmd = _get_create_command(
+            self.emulated_image, params.get("emulated_image_size"), self.create_cmd
+        )
```

Comment:
IMO, this should be executed within the condition `self.iscsi_backend == "fileio"` since it seems not available for the `block` iscsi backend.
Maybe you have your own intention here. Please let me know, thanks.

Reply:
The create_cmd is meaningless in the present code. 
Why not create directly?
My point is one API wrap "process.system(self.create_cmd) " .
It follows the current business, but It removes detection operations. 
You may understand that the API name  is  create_backstore(xxxxx)

BTW, the current solution is LGTM